### ISSUE TITLE
Beacon Post-Calibration Report

### ISSUE THREAD

**Issue by HagenFritz** [intelligent-environments-lab/utx000#18]
# Report on Post-Calibration
After downloading the data, it would be interesting to see how well the beacons are performing relative to a reference-grade monitor. Thus this report would focus on the difference in readings between the beacons and the references:

## Beacon Post-Calibration
Important aspects of the report to include:
- Time series/percent-difference plots for the various environmental variables we are able to measure
- Analyses comparing the degree of difference

Additional features might include
- Any empirical "fudge factors" we can apply to readings to improve the measurements

**Comment by HagenFritz** [comment]
CO2 monitor set up at the test house

Waiting on APS to get fixed

Need to email Atila about the NO2 sensor

**Comment by HagenFritz** [comment]
APS will come back around 11/05

NO2 sensor needs to be calibrated - Mengjia is working on this

**Comment by HagenFritz** [comment]
APS is back, NO2 sensor is calibrated, just waiting on approval to get back into the test house

**Comment by HagenFritz** [comment]
Everything is set up! Just need to schedule a time to calibrate

**Comment by HagenFritz** [comment]
See captain's log [here](https://github.com/intelligent-environments-lab/utx000/tree/master/reports/post_calibration) for updates on the current calibration efforts.

**Comment by HagenFritz** [comment]
Linear Regression models seem to work nicely. Future work might entail using more advanced methods, but this should work for the time-being.
